.github/workflows/test.yaml

```diff
@@ -0,0 +1,52 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  IP: 127.0.0.1
+
+jobs:
+  test:
```

Comment:
one last request, can we split it into two jobs `test-swarm` and `test-k8s` so that they can be run in parallel?  it otherwise looks great

sorry, i didn't ask about it before i had to read the docs but had an issue at work that prevented me from commenting sooner

Reply:
thank you lucas done.